### DISCUSSION TITLE
CPS to use ?

### DISCUSSION THREAD

**Discussion by OE1MWW** [egzumer/uv-k5-firmware-custom#33] [Q&A] [answered]
Hi all,
I am new to this group & project.
Can I use the "UV-K5(8) Programming Software-Neutral\cps_V1.0.40_免安装版" to load my repeater list ?
73s

**Comment by egzumer** [comment]
Should work ok, but I would recommend chirp

**Comment by ErikS-web** [comment, marked as answer]
Added some extra info at the wiki under [How2 use Chirp with this radio](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Radio-operation#how2-use-chirp-with-this-radio)
